Probesys/bileto
# Imp/redesign tickets list

## Related issue(s)



https://github.com/Probesys/bileto/issues/209

## Changes



- Remove year from dates if it's the same as current year
    - add two filters to display dates which are able to show the year only when needed (i.e. when the given date is different from the current one)
- Add more variant of badges and remove specific code related to them (i.e. on the "show" page of tickets)
- Add a `.row--middle` class to center items in their container
- Change the tables of tickets by custom lists (more appealing + more adaptable to small screens)
- Remove the style related to tables (not used anymore)

## How to test manually



- go to the home page, "tickets" and "organization tickets" pages → check the lists look right, both on big and small screens

## Checklist
